ansibleplaybookgrapher/parser.py

```diff
@@ -234,6 +237,11 @@ def parse(self, *args, **kwargs) -> PlaybookNode:
                 if role.from_include:
                     continue
 
+                # Don't show roles, which are set in --exclude-roles option
+                if self.exclude_roles:
+                    if role.get_name() in self.exclude_roles:
+                        continue
```

Comment:
I think we can simplify this by removing the condition (alongside the other suggestion above). 

```suggestion
                if role.get_name() in self.exclude_roles:
                    continue
```

Same for the next one.